src/components/NewLocationPage/ChartFooter/AddedMetricChartFooter.tsx

```diff
@@ -1,3 +1,5 @@
+/** Chart footer for newly added chart metrics (deaths, hospitalizations, ICU hospitalizations) */
```

Comment:
nit: Heh, again "newly added" is relevant right now but won't make much sense to somebody looking at this code in the future. 😆   But I guess we have `Added` in the filename as well, so a broader cleanup is in order eventually.

Reply:
yeah i agree--something more timeless/descriptive is the way to go. just wasnt sure what to call these files/components/functions so went with added/newly added for the time being 😬